### PR TITLE
Rework PDF script

### PR DESCRIPTION
**Pull request by zdroid**
Highlights:
- matched the coding style for all files in `scripts/pdf/` with the settings provided in `.editorconfig`
- removed duplicate CSS, by using `basic.css` as basic CSS (:laughing:) and themes as layers on top of it
- matched CSS and HTML with general standards for those
- added PDF-related files to `.gitignore`

There may be more to do:
- remove the color themes as they really seem unesthetic
- add a note on how to make own styles

(although I'd keep the last two for some future PR)

### CODE REVIEW

File: scripts/pdf/render.py [hunk 1]
```diff
@@ -15,108 +15,111 @@
...
 
-    oslist.sort()
+  oslist.sort()
 
-    #Required strings to create intermediate HTML files
-    header = "<html><head><link rel=stylesheet type=text/css href=" + colorscheme + ".css></head><body>\n"
-    footer = "</body></html>"
-    title_content = "<h1 class=titlemain>tldr pages</h1><h4 class=titlesub>Simplified and community driven man pages</h4></body></html>"
+  # Required strings to create intermediate HTML files
+  header = '<!DOCTYPE html><html><head><link rel="stylesheet" href="basic.css">'
```
Comment (agnivade) [inline scripts/pdf/render.py]: Wait, so we weren't using the basic.css at all ? But the file seems to be already there ?

Comment (zdroid) [reply scripts/pdf/render.py]: We were, but it was treated as a color scheme. So every scheme was included completely, as a file for itself. That way we had a lot of duplicate styles in 3 files. Another solution as I said in the PR description is to keep it as one file, but to remove our color schemes as they're really bad-looking.

Comment (agnivade) [reply scripts/pdf/render.py]: Nope, this looks fine. I will review when I have some time.

**Comment by sbrl** [comment]
Nice one, @zdroid!

Suggestion: Create a `requirements.txt` file, so that you can do `pip3 install -r requirements.txt`

**Comment by zdroid** [comment]
@sbrl here you go!

File: scripts/pdf/render.py (hunk 1 above)
Comment (agnivade) [inline scripts/pdf/render.py]: While we are here, let's also add a "meta charset=utf-8"

**Pull request merged**